helmsman/Makefile

```diff
@@ -3,7 +3,7 @@ SHELL            := /bin/bash
 MAKEFILES_DIR    := ../@bin/makefiles
 
 DOCKER_ORG 		:= binbash
-DOCKER_IMAGE 	:= helmsman
+DOCKER_IMG_NAME := helmsman
 DOCKER_TAG 		:= v3.4.3-helm-v3.2.1
```

Comment:
@diego-ojeda-binbash I think we could update https://github.com/binbashar/le-docker-images/blob/master/helmsman/Dockerfile ->  
`ARG BASE_HELMSMAN_IMAGE=praqma/helmsman:v3.4.3-helm-v3.2.1`

with 

```
ARG DOCKER_TAG
ARG BASE_HELMSMAN_IMAGE=praqma/helmsman:${DOCKER_TAG}
```
To avoid harcoding the value at the Dockerfile

Reply:
@exequielrafaela That actually isn't a hardcoded value, it is a default value in case one is not provided.
I mean, that argument is mandatory, you must pass it or the build will fail. So you pass it from the Makefile that triggers the docker build command: https://github.com/binbashar/le-docker-images/blob/helmsman/helmsman/Makefile#L20
However, kind of related to your point, I can rather update said Makefile like this:
```
BASE_HELMSMAN_IMAGE	:= praqma/helmsman:${DOCKER_TAG}
```